__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -143,16 +143,12 @@ describe('TaskDetails Page', () => {
             }
         );
         await waitFor(() => {
-            const button = screen.getByLabelText('Open GitHub Issue');
-            const originalOpen = window.open;
-            const mockOpen = jest.fn();
-            window.open = mockOpen;
-            fireEvent.click(button);
-            expect(mockOpen).toHaveBeenCalledWith(
-                'https://www.sampleGithubUrl.com',
-                '_blank'
+            const link = screen.getByText('Issue #000');
+            fireEvent.click(link);
+            expect(link).toHaveAttribute(
+                'href',
+                'https://github.com/sampleOrg/sampleRepo/issues/000'
```

Comment:
why the url is camelcase?

Reply:
No specific reason; casing doesn't matter for HTML URLs; it's just for better readability.

Reply:
i have addressed